source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,266 @@
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+.. _sort_by_relevance:
+
+Sort by Relevance
+~~~~~~~~~~~~~~~~~
+
+The text search assigns a numerical text score to indicate how relevant
+each result matches the search string. Often, you want to know which
+results are the most relevant.  
+
+To sort by relevance, you use the :doc:`Projections
+</fundamentals/builders/projections>`  and :doc:`Sorts
+</fundamentals/builders/sort>` builders.
+
+First, create a project document to retrieve a text score by using the
+``Projections.metaTextScore()`` method. Upon passing this document to the
+``projection()`` method generates a new field with the specified name
+and returns the documents text score. 
+
+Then, specify a sort criteria to sort by the text score by using the
+``Sorts.metaTextScore()`` method. Upon passing this to the ``sort()``
+method, MongoDB sorts the results from the most to least relevant. By
+default, the method returns the results unsorted. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the term
+"Fast". 
+
+To view them in order of relevance to the search, perform the following
+actions: 
+
+- Project the text scores of the query results as the ``score`` field
+- Sort the results by text score (most relevant match first)
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin scoreExample
+   :end-before: end scoreExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"], "score": 0.75 }
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"], "score": 0.625 }
+   
+The movie with the ``_id`` value of 2 is the most relevant to our
+search. Although both documents contain what the search phrase we
+were looking for, MongoDB then ranks based on the string length of the
+``title`` field. 
+
+For more information, see the following documentation:
+
+- :java-docs:`Projections.metaTextScore() <apidocs/mongodb-driver-core/com/mongodb/client/model/Projections.html#metaTextScore(java.lang.String)>` API documentation
+- :java-docs:`Sorts.metaTextScore() <apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html#metaTextScore(java.lang.String)>` API documentation
+- :manual:`Text score metadata </reference/operator/aggregation/meta/#mongodb-expression-exp.-meta>` Server Manual Entry 
+- :manual:`Text search score </reference/operator/query/text/#return-the-text-search-score/>` Server Manual Entry 
```

Comment:
Shouldn't these be title case if you're using title case for "Server Manual Entry"?